### PR TITLE
Add examples

### PR DESCRIPTION
**Pull request by OR13**
adds examples of did documents with verification methods that are defined in the did core context.
adds examples of verifiable credentials leveraging those verification methods.
adds examples of encryption using JWE leveraging verification methods.
adds placeholder for ZKPs (may get removed if we can't provide examples).

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/pull/377.html" title="Last updated on Aug 20, 2020, 9:45 PM UTC (3f129ae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/did-core/377/4dedfe5...3f129ae.html" title="Last updated on Aug 20, 2020, 9:45 PM UTC (3f129ae)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4007,6 +4007,325 @@ <h2>
...
+    </section>
+
+    <section class="informative">
+      <h2>
+        Zero Knowledge Proofs
+      </h2>
+      <p class="issue">
+        Add BBS+ / HyperLedger Indy examples here, or remove section.
+      </p>
+    </section>
```
Comment (msporny) [inline index.html:4325]:
```suggestion
```

No need to put in a place holder. Just do a separate PR once we have an example.

File: index.html [hunk 2]
```diff
@@ -4007,6 +4007,325 @@ <h2>
...
+            }
+          ]
+      }
+      </pre>
+
+      <pre class="example" title="DID Document with many different verification methods">
+        {
+          "@context": "https://www.w3.org/ns/did/v1",
+          "id": "did:example:123",
+          "publicKey": [
```
Comment (dlongley) [inline index.html:4071]:
```suggestion
          "verificationMethod": [
```

File: index.html [hunk 3]
```diff
@@ -4007,6 +4007,317 @@ <h2>
...
+        DID Documents
+      </h2>
+
+      <p>
+        See <a href="https://www.w3.org/TR/did-spec-registries/#verification-method-types">did-spec-registries</a> for optional extensions and other verifcation method types.
+      </p>
+
+      <p class="note">
+        These examples are for information purposes only, 
+        it is considered a best practice to avoid using the same verification method for multiple purposes.
```
Comment (kdenhartog) [inline index.html:4025]: Great to see this statement in here! This will prove incredibly valuable I expect.

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.
